### PR TITLE
Only allow one info window at a time.

### PR DESCRIPTION
**Pull request by joshluongo**
## Description
I've made it so you can only open one info window at a time. This also fixes and issue where you can open multiple info windows for the same marker.

I've also added a convince function `clearInfoWindows()` to close all open info windows.

## Motivation and Context
On a phone the screen get's very crowded with multiple info windows open. If the user keeps tapping the marker multiple times then it can be frustrating to close all of the windows.

## How Has This Been Tested?
This is a minor change but I've tested it on iOS and Safari on Mac OS X.

## Screenshots (if appropriate):
_I can't show the change in a screenshot_

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by brycedorn** [comment]
thanks for addressing this! was bugging me 👼 

rather than iterating through everything in `map_data` though, how about adding `info_windows` as a hash in `map_data`? this would be a lot more efficient, and would make checking if a pokemon has an `info_window` already open much easier

File: static/map.js [hunk 1]
```diff
@@ -619,8 +619,37 @@ function clearSelection() {
...
+
+    $.each(map_data.pokestops, function(key, value) {
+        // Close all windows
+        value.marker.infoWindow.close();
+    });
+};
+
 function addListeners(marker) {
     marker.addListener('click', function() {
+    	clearInfoWindows();
```
Comment (brycedorn) [inline static/map.js]: also indentation slightly off here^ 🔍

**Comment by joshluongo** [comment]
@brycedorn The indentation is because I edited it via the GitHub editor 🙄. I think placing it in a hash map is a good solution. I'll update my branch with that change (and I'll fix the indentation 😉) when I get a chance.

**Comment by joshluongo** [comment]
I'm going to rewrite my changes and submit a new PR.

**Pull request closed without merging**